src/MethodProxies-Tests/MpMethodProxyTest.class.st

```diff
@@ -56,6 +56,33 @@ MpMethodProxyTest >> tearDown [
 	super tearDown
 ]
 
+{ #category : #'tests - safety' }
+MpMethodProxyTest >> testCanRunConcurrently [
+
+	| mp1 mp2 mpTrigger1 mpTrigger2 handlerTrigger1 handlerTrigger2 concurrencyClass testSemaphore |
+	
+	concurrencyClass := MpConcurrencyClass new.
```

Comment:
The class is already a class (no need to name it "Class").
And its instance is an instance, not a class :)

Let's rename them a bit. Maybe they can be called something in the order of `MpTestConcurrentSharedObject`?